docs/zh_cn/user_guides/inference.md

```diff
@@ -1,237 +1,418 @@
 
-```shell
-python demo/inpainting_demo.py \
-    ${CONFIG_FILE} \
-    ${CHECKPOINT_FILE} \
-    ${MASKED_IMAGE_FILE} \
-    ${MASK_FILE} \
-    ${SAVE_FILE} \
-    [--imshow] \
-    [--device ${GPU_ID}]
+# 创建MMagicInferencer实例
+editor = MMagicInferencer('pix2pix')
+# 推理图片.需要输入图片路径与输出图片路径
+results = editor.infer(img='../resources/input/translation/gt_mask_0.png', result_out_dir='../resources/output/translation/tutorial_translation_pix2pix_res.jpg')
 ```
 
-如果指定了 --imshow ，演示程序将使用 opencv 显示图像。例子：
+MMagic支持各种基础生成模型，包括无条件生成对抗网络（GANs）、条件GANs、扩散模型等。
 
-```shell
-python demo/inpainting_demo.py \
-    configs/global_local/gl_8xb12_celeba-256x256.py \
-    https://download.openmmlab.com/mmediting/inpainting/global_local/gl_256x256_8x12_celeba_20200619-5af0493f.pth \
-    tests/data/inpainting/celeba_test.png \
-    tests/data/inpainting/bbox_mask.png \
-    tests/data/inpainting/inpainting_celeba.png
+MMagic 同样支持多种应用，包括：
+
+文生图、图生图的转换、3D感知生成、图像超分、视频超分、视频帧插值、图像修补、图像抠图、图像恢复、图像上色、图像生成等。
+
+在本节中，我们将详细说明如何使用我们预训练的模型进行操作。
+
+
+
+- 教程3: 使用预训练模型推理
+  - [准备一些图片或者视频用于推理](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Prepare-some-images-or-videos-for-inference)
+  - 生成模型
+    - [无条件生成对抗网络 (GANs)](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Unconditional-Generative-Adversarial-Networks-(GANs))
+    - [条件生成对抗网络(GANs)](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Conditional-Generative-Adversarial-Networks-(GANs))
+    - [扩散模型](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Diffusion-Models)
+  - 应用
+    - [文生图](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Text-to-Image)
+    - [图生图的转换](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Image-to-image-translation)
+    - [3D感知生成](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#3D-aware-generation)
+    - [图像超分](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Image-super-resolution)
+    - [视频超分](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Video-super-resolution)
+    - [视频帧插值](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/Video-frame-interpolation)
+    - [图像修补](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Image-inpainting)
+    - [图像抠图](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Image-matting)
+    - [图像恢复](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Image-restoration)
+    - [图像上色](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Image-colorization)
+- [以前的版本](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/inference.md#Previous-Versions)
+
+
+
+
+
+
+
+
+
+
```

Comment:
Too many blank lines in this section. Please install and run `pre-commit hook` to fix this kind of problem.